content/en/docs/reference/command-line-tools-reference/feature-gates/pod-lifecycle-sleep-action.md

```diff
@@ -9,5 +9,8 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.29"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.30"
```

Comment:
Adding `toVersion` to the previous stage would enhance consistency with other feature gate files. 
```suggestion
    fromVersion: "1.29"
    toVersion: "1.29"
  - stage: beta 
    defaultValue: true
    fromVersion: "1.30"
```

Reply:
Yes, I missed this🤦. Thanks.